Header Header
uint8 id
int32 goal_pos # in radians
int32 curr_pos # in radians
